AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -589,20 +590,33 @@ - (void)openURLInAuthenticationSessionWith:(NSURL *)url API_AVAILABLE(ios(11)) {
     if (callbackUrl) {
       [strongSelf openURL:callbackUrl];
     }
+    [[MSUtility sharedApp] endBackgroundTask:backgroundAuthSessionTask];
   };
   SFAuthenticationSession *session = [[SFAuthenticationSession alloc] initWithURL:url
                                                                 callbackURLScheme:callbackUrlScheme
                                                                 completionHandler:authCompletionBlock];
 
+  if (self.authenticationSession != nil) {
+
+    // Calling 'start' on an existing session crashes the application - clear session.
+    [self clearAuthenticationSession];
```

Comment:
You can just call cancel on the current session right?

Reply:
If we retry to assign the value to authSession without removing the reference we will get a fatal error. So we also need to remove the link.